pkg/rtc/mediatrack.go

```diff
@@ -226,6 +226,7 @@ func (t *MediaTrack) AddSubscriber(sub types.Participant) error {
 		return err
 	}
 	subTrack := NewSubscribedTrack(SubscribedTrackParams{
+		PublisherID:       t.params.ParticipantID,
```

Comment:
Once gain, sid vs identity issues. Looks like we can swap out `PublisherID` for `PublisherIdentity` in `SubscribedTrack`. But, have left `PublisherIdentity` in there for now. Please let me know if it can be cleaned out and I will add a commit to this PR to clean that out.